common-job/src/test/java/org/uniprot/store/reader/publications/CommunityMappedReferenceConverterTest.java

```diff
@@ -64,4 +64,21 @@ void convertingSingleWordHasNoFullStop() throws IOException {
         assertThat(communityAnnotation.getComment(), is("Peas"));
         assertThat(communityAnnotation.getDisease(), is("This is a disease."));
     }
+
```

Comment:
Could you please add a test case to verify if the category doesn't match to any one our categories? In that case, we should get Unclassified category.

Reply:
I will do this test in MappedReferencesToPublicationDocumentBuilderConverter.java, because we might not have a category for community, but it could come from computationally mapped. We have to verify after we group all mappedReferences